frame/system/benchmarking/src/lib.rs

```diff
@@ -48,9 +48,12 @@ benchmarks! {
 	set_heap_pages {
 	}: _(RawOrigin::Root, Default::default())
 
-	// `set_code` was not benchmarked because it is pretty hard to come up with a real
-	// Wasm runtime to test the upgrade with. But this is okay because we will make
-	// `set_code` take a full block anyway.
+	set_code {
+		let runtime_blob = include_bytes!("../res/kitchensink_runtime.compact.compressed.wasm");
+	}: _(RawOrigin::Root, runtime_blob.to_vec())
```

Comment:
```suggestion
		let runtime_blob = include_bytes!("../res/kitchensink_runtime.compact.compressed.wasm").to_vec();
	}: _(RawOrigin::Root, runtime_blob)
```

Let's create the vector outside of the loop.